app/src/containers/FeedData.js

```diff
@@ -12,12 +27,37 @@ const query = gql`
   }
 `;
 
+const subscriptionQuery = gql`
+  subscription {
+    feedSubscription {
+      mutation
+      node {
+        id
+        message
+        createdAt
+      }
+      previousValues {
+        id
+        message
+      }
+    }
+  }
+`;
+
 const enhanced = compose(
   graphql(query),
   withProps(({ data: { loading, feed } }) => ({
     loading: loading,
     comments: feed
-  }))
+  })),
+  lifecycle({
+    componentWillMount() {
+      this.props.data.subscribeToMore({
```

Comment:
`subscribeToMore` magic updates feed as new comments appear in real-time.. this is so awesome!